### ISSUE TITLE
explain result error

### ISSUE THREAD

**Issue by FlyingMao** [actiontech/dble#2042]
* **dble version：**  
dble-2.20.04.0-7b5f749b8fbea5443c6e21cb520b2c2afbafa48d-20200519034453
* **preconditions ：**  
```
CREATE TABLE `o_dept` (
  `DEPT_NO` varchar(16) NOT NULL,
  `ORG_NO` varchar(16) NOT NULL,
  `NAME` varchar(256) DEFAULT NULL,
  PRIMARY KEY (`DEPT_NO`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;

CREATE TABLE `o_org` (
  `ORG_NO` varchar(16) NOT NULL,
  `ORG_NAME` varchar(256) DEFAULT NULL,
  `P_ORG_NO` varchar(16) DEFAULT NULL,
  PRIMARY KEY (`ORG_NO`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;

CREATE TABLE `p_sys_user` (
  `SYS_USER_NAME` varchar(30) NOT NULL,
  `DEPT_NO` varchar(16) DEFAULT NULL,
  `ORG_NO` varchar(16) NOT NULL,
  `USER_NAME` varchar(64) DEFAULT NULL,
  `PWD` varchar(256) NOT NULL,
  `CUR_STATUS_CODE` varchar(8) DEFAULT NULL,
  `ADMIN_FLAG` decimal(3,0) DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
```
* **configs：** 

**schema.xml**  

```
<table name="o_dept" dataNode="dn1" />
 <table name="p_sys_user" dataNode="dn1"/>
<table name="o_org" dataNode="dn1,dn2" type="global"/>
```

**rule.xml**  

```


```

**server.xml**  

```


```

* **steps：**    
    step1.  
```
explain SELECT
	a.sys_user_name AS sysUserName,
	d.org_no AS orgNo,
	d.org_name AS orgName,
	c.dept_no AS deptNo,
	c. NAME AS deptName,
	a.user_name AS userName,
	a.cur_status_code AS curStatusCode,
	a.admin_flag AS adminFlag
FROM
	p_sys_user a
LEFT JOIN o_dept c ON a.dept_no = c.dept_no,
 o_org d
WHERE
	a.org_no = d.org_no
AND a.org_no IN (SELECT org_no FROM o_org)
AND (
	a.cur_status_code IS NULL
	OR a.cur_status_code <> '03'
)
```
* **expect result：**  
    1. 
```
| DATA_NODE | TYPE     | SQL/REF                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| dn1_0     | BASE SQL | select `a`.`sys_user_name` as `sysUserName`,`d`.`org_no` as `orgNo`,`d`.`org_name` as `orgName`,`c`.`dept_no`s `userName`,`a`.`cur_status_code` as `curStatusCode`,`a`.`admin_flag` as `adminFlag` from  (  (  `p_sys_user` `a` left join  `o_dept``d` )  join (select  distinct `o_org`.`org_no` as `autoalias_scalar` from  `o_org`) autoalias_o_org where `a`.`org_no` = `d`.`org_no`  and (a.cur_status_code IS NULL OR `a`.`cur_status_code` <> '03') |
| merge_1   | MERGE    | dn1_0       ****
```
* **real result：**  
    1.
```
| DATA_NODE | TYPE     | SQL/REF                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| dn1_0     | BASE SQL | select `a`.`sys_user_name` as `sysUserName`,`d`.`org_no` as `orgNo`,`d`.`org_name` as `orgName`,`c`.`dept_no`s `userName`,`a`.`cur_status_code` as `curStatusCode`,`a`.`admin_flag` as `adminFlag` from  (  (  `p_sys_user` `a` left join  `o_dept``d` )  join (select  distinct `o_org`.`org_no` as `autoalias_scalar` from  `o_org`) autoalias_o_org where `a`.`org_no` = `d`.`org_no`  and (a.cur_status_code IS NULL) OR (`a`.`cur_status_code` <> '03') |
| merge_1   | MERGE    | dn1_0                                           
```
* **supplements：**  
    1.

**Comment by yanhuqing666** [comment]
now fix only in  2.20.04.99

**Comment by wjl1619** [comment]
verified on version:  5.7.21-dble-9.9.9.9-60d726b625c288e1d5715d41c90fbab06ab758ce-20201111094506
